src/linker/Linker.Steps/TrimUsingEntryPoint.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+	public class TrimUsingEntryPoint : BaseStep
```

Comment:
I think it would be better to call this `RootEntryPointStep` or something like that... "trim using entry point" feels weird - it acts on a single assembly, but this step doesn't actually do the trimming - so it's confusing.

Reply:
Changed the approach in follow up PR